app/views/layouts/_header.html.erb

```diff
@@ -24,6 +24,7 @@
   <%= render(HeaderComponent.new(
     classes: "app-header--#{HostingEnvironment.environment_name} app-header--full-border app-header--wide-logo",
     product_name: service_name,
+    homepage_url: service_link,
```

Comment:
Should we do the same for support given the header is similar to manage.

Reply:
good idea!